CMakeLists_files.cmake

```diff
@@ -301,6 +301,7 @@ list (APPEND TEST_SOURCE_FILES
       tests/test_cubic.cpp
       tests/test_messagelimiter.cpp
       tests/test_nonuniformtablelinear.cpp
+      tests/test_OpmInputError_format.cpp
```

Comment:
I guess this file is not added to the commit

Reply:
Yep, that slipped through.